cclib/parser/molcasparser.py

```diff
@@ -222,36 +222,19 @@ def extract(self, inputfile, line):
         #  Threshold at which QNR/C2DIIS is turned on 0.75E-01
         #  Threshold for Norm(delta) (QNR/C2DIIS)     0.20E-04
         if line[:34] == '++    Optimization specifications:':
-
-            scftargets = []
-            while not line[6:37] == 'Threshold for SCF energy change':
-                line = next(inputfile)
-
-            if line[6:37] == 'Threshold for SCF energy change':
-                target = float(line.split()[-1])
-                scftargets.append(target)
-
-            line = next(inputfile)
-
-            if line[6:34] == 'Threshold for density matrix':
-                target = float(line.split()[-1])
-                scftargets.append(target)
-
-            line = next(inputfile)
-
-            if line[6:31] == 'Threshold for Fock matrix':
-                target = float(line.split()[-1])
-                scftargets.append(target)
-
-            self.skip_lines(inputfile, ['Threshold for linear dependence', 'Threshold at which DIIS is turned on', \
-                                         'Threshold at which QNR/C2DIIS is turned on'])
+            self.skip_lines(inputfile, ['d', 'b'])
             line = next(inputfile)
-
-            if line[6:31] == 'Threshold for Norm(delta)':
-                target = float(line.split()[-1])
-                scftargets.append(target)
-
-            self.append_attribute('scftargets', scftargets)
+            if line.strip().startswith('SCF'):
+                scftargets = []
+                self.skip_lines(inputfile,
+                    ['Minimized', 'Number', 'Maximum', 'Maximum'])
+                lines = [next(inputfile) for i in range(7)]
+                targets = ['Threshold for SCF energy change', 'Threshold for density matrix', 'Threshold for Fock matrix',
+                                    'Threshold for Norm(delta)']
```

Comment:
Something is wrong with the indentation here. Also, the top line is too long. Try to either do
```python
targets = [
    'apple',
    'banana',
    'pear',
]
```
or
```python
targets = ['apple',
           'banana',
           'pear']
```
I don't think we've standardized on one or the other.